docs/devGuide/settingUp.md

```diff
@@ -35,14 +35,24 @@ We recommend the **WebStorm IDE** for working with MarkBind code.
 
 1. **Fork and clone** the MarkBind repo.
 1. **To bind your cloned version of MarkBind to your console** (instead of the released version of MarkBind), run `npm link` in the `packages/cli` folder of the cloned repo first.
-
-   <box type="tip" seamless>
-
-   To go back to the released version of MarkBind, run `npm un -g markbind-cli`, followed by `npm i -g markbind-cli`
-   </box>
 1. **Install dependencies** by running <popover content="Under the hood, this calls `npm ci` and `lerna bootstrap`">`npm run setup`</popover> in the **root folder** of your cloned repo.
-  <box type="info" seamless>
-  MarkBind uses <md>[lerna](https://github.com/lerna/lerna)</md>, a popular multi-package development tool, to manage it's dependencies. It is essentially a high level wrapper over node and npm's functionalities
-  </box>
 
 1. **Congratulations!** Now you ready to start modifying MarkBind code.
+
+### Common setup questions
+
+1. Does MarkBind work with all operating systems?
+
+   Yes! We support all operating systems. However, Windows 11 users might have to run `npm install` after `npm run setup` as some packages may have incompatible versions.
+
+1. How does Markbind manage dependencies?
+
+   MarkBind uses [lerna](https://github.com/lerna/lerna), a popular multi-package development tool, to manage it's dependencies. It is essentially a high level wrapper over node and npm's functionalities.
+
+1. How do I move back to the released version of MarkBind?
+
+   To go back to the released version of MarkBind, run `npm run -g markbind-cli`, followed by `npm i -g markbind-cli`.
+
+1. Some of my front-end components are not working as expected.
+
+   Try running `npm run build:web` to rebuild the front-end components as these changes are only made when a new version of MarkBind is released.
```

Comment:
Maybe mentioning `markbind serve -d` could be helpful too? These two solutions are actually already mentioned in [this part](Fix tipBox content overflow with min-width) of Workflow by the way, we can also refer to it if they're developing frontend features.

Reply:
> Maybe mentioning `markbind serve -d` could be helpful too? These two solutions are actually already mentioned in [this part](https://markbind.org/devdocs/devGuide/workflow.html#editing-frontend-features) of Workflow by the way, we can also refer to it if they're developing frontend features.

Yeah I noticed it as well, I decided to add it to this setting up section as I felt it might be relevant since building the frontend features might be required especially if there was a recent push to those components and the .min files have not been updated yet. I will add a link to the relevant sections though. Thanks for the feedback.